### ISSUE TITLE
[Linux] AttributeError: 'NoneType' object has no attribute 'text'

### ISSUE THREAD

**Issue by adro79** [jchanvfx/NodeGraphQt#369]
Hi there, I'm currently testing [QuiltiX](https://github.com/PrismPipeline/QuiltiX) on Linux.

I'm using nodegraphqt 0.6.16 with python 3.11 on Arch Linux.

This is one of the errors I encountered:

```
File "/usr/lib/python3.11/site-packages/NodeGraphQt/base/menu.py", line 247, in __init__
    self._name = qaction.text()
                 ^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'text'
```

I can avoid this error by commenting the line.

**Comment by jchanvfx** [comment]
might be related to this issue https://github.com/jchanvfx/NodeGraphQt/issues/298

**Comment by jchanvfx** [comment]
Hi @adro79, 

I've just released `v0.6.18` can you try this version.
https://github.com/jchanvfx/NodeGraphQt/releases/tag/v0.6.18

Cheers,
J
